sdk/eventhub/Azure.Messaging.EventHubs/CONTRIBUTING.md

```diff
@@ -1,19 +1,70 @@
 ﻿# Contributing 
 
-Coming Soon...
+Thank you for your interest in contributing to the Event Hubs client library.  As an open source effort, we're excited to welcome feedback and contributions from the community.  A great first step in sharing your thoughts and understanding where help is needed would be to take a look at the [open issues](https://github.com/Azure/azure-sdk-for-net/issues?q=is%3Aopen+is%3Aissue+label%3AClient+label%3A%22Event+Hubs%22).
 
-For more details about contributing to the Azure Event Hubs client libraries or our other .NET SDKs, please refer to: [Azure SDK for .NET Contributing Guide](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md)
+ Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use any contribution that you make. For details, visit https://cla.microsoft.com.
 
-### Live test information
+## Code of conduct
 
-### Target frameworks
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-For information about the target frameworks of the Azure Event Hubs client library, please refer to the [Target Frameworks](https://github.com/azure/azure-sdk-for-net#target-frameworks) of the Microsoft Azure SDK for .NET.  
+## Getting started
 
-### Versioning information
+Before working on a contribution, it would be beneficial to familiarize yourself with the process and guidelines used for the Azure SDKs so that your submission is consistent with the project standards and is ready to be accepted with fewer changes requested.   In particular, it is recommended to review:
 
-The Azure Event Hubs client library uses [the semantic versioning scheme.](http://semver.org/)
+  - [Azure SDK README](https://github.com/Azure/azure-sdk), to learn more about the overall project and processes used.
+  - [Azure SDK Design Guidelines](https://azuresdkspecs.z5.web.core.windows.net/DesignGuidelines.html#general-documentation), to understand the general guidelines for the SDKs across all languages and platforms
+  - [Azure SDK Design Guidelines for .NET](https://azuresdkspecs.z5.web.core.windows.net/DotNetSpec.html), to understand the guidelines specific to the Azure SDKs for .NET.
```

Comment:
I wonder if until here, there should be generic to the whole repo and not per project.

@kaerm

Reply:
I don't disagree;   if you look at the contributing guide for the .NET repository all-up, it's very focused on the management libraries and barely mentions the client libraries in passing.

It also isn't very cohesive or comprehensive, if I'm being honest.   I'd love to see that be something that we were comfortable pointing to for general guidance.

Reply:
Yeah.... I really like the structure and will like to reference it for KV... for now we can leave here and maybe later move it to that general Readme?

Reply:
@AlexGhiondea:  Do you know who would be considered the "owner" of the main contributing guide for the .NET repository and if there are plans to give that some love?   Mariana makes some very good points about not repeating the same content per-client library...

Reply:
@loarabia @kaerm should help us figure out the main readme file.